.dockerignore

```diff
@@ -0,0 +1,11 @@
+.dockerignore
+docker
+Dockerfile
+docker-compose.yml
+.git
+.gitignore
+frontend/node_modules
+frontend/cypress/mochawesome
+frontend/mochawesome-report
+frontend/mochawesome.json
+venv
```

Comment:
newline? (same the other docker files below)

Reply:
Strange, I'm seeing a new line in my editor, `atom`